opg-mongodb-1604/docker/my_init/99-setup-mongo

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/opt/configure_db.py &
```

Comment:
pushing this into the background means the server can start up without forming a cluster and the error is written belatedly to the logs, which do not seem to be sent to logstash